manifests/i/iQIYI/iQIYI/8.6.138.3213/iQIYI.iQIYI.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate
+
+PackageIdentifier: iQIYI.iQIYI
+PackageVersion: 8.6.138.3213
+PackageLocale: en-US
+Publisher: 北京爱奇艺科技有限公司
+PublisherUrl: https://www.iqiyi.com
+PrivacyUrl: https://privacy.iqiyi.com/policies
+PackageName: iQIYI

```

Comment:
```suggestion
Publisher: 爱奇艺
PublisherUrl: https://www.iqiyi.com
PrivacyUrl: https://privacy.iqiyi.com/policies
PackageName: 爱奇艺
```

Reply:
Quick question: Does it matter what is written in additional locales under PackageName and Publisher as long as it is correct in the defaultLocale?

Reply:
> Quick question: Does it matter what is written in additional locales under PackageName and Publisher as long as it is correct in the defaultLocale?

I believe it acts as additional matching metadata.

So if the English installer installs `iQIYI` it would match and if the Chinese installer installs `爱奇艺` it would also match.
Another example is if we used the whole Firefox `PackageName` it would match for `Mozilla Firefox (en-US)` and `Mozilla Firefox (ch-CN)`

However we don't seem to a lot of the very localized software in the repo.

Reply:
Sorry but as the app owner - iQIYI, Inc (a.k.a 爱奇艺, NASDAQ: IQ) - required, the publisher name should be 北京爱奇艺科技有限公司 (Beijing iQIYI Science & Technology Co., Ltd.) which is the commercial entity registered. This is also applied for the UWP app of iQIYI that you can have a check by following.
https://www.microsoft.com/en-us/p/%E7%88%B1%E5%A5%87%E8%89%BA-%E6%82%A6%E4%BA%AB%E5%93%81%E8%B4%A8/9nblggh5wxnw

Reply:
This info has to match the entry in the Add/Remove programs list to allow updates and compatibility with the winget commands.

Reply:
We do not make the rules, we just enforce the rules.
![image](https://user-images.githubusercontent.com/15158490/127988291-62921b44-9dfb-4066-b667-921fd5b53e50.png)
This Metadata needs to match what is show in Add/Remove Programs as quhxl said, this information is stored in the Registry at `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\PPStream`

Reply:
OK, so I will update the publisher after we update the metadata in Add/Remove Programs. And name the publisher to 爱奇艺 for current version.